doc_classes/TrainBrake.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="TrainBrake" inherits="TrainPart" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/godotengine/godot/master/doc/class.xsd">
+	<brief_description>
```

Comment:
Wcięcia w xml nie mają 4 spaces. O ile są w ogóle spacje, bo tego nie widzę. Do tej pory mieliśmy używać 4  spacji. Jeśli zmieniła się konwencja, to trzeba zmienić wszędzie.